test/e2e/tests/signature/eth-sign.spec.js

```diff
@@ -21,7 +21,7 @@ describe('Eth sign', function () {
         await driver.delay(1000);
         const ethSignButton = await driver.findElement('#ethSign');
         const exceptionString =
-          'ERROR: THE METHOD "ETH_SIGN" DOES NOT EXIST / IS NOT AVAILABLE.';
+          'Error: The method "eth_sign" does not exist / is not available.';
```

Comment:
Given that we are not making live requests to get fonts and icons, the format for the text is changed, now using lower case:

![Screenshot from 2025-06-06 14-26-36](https://github.com/user-attachments/assets/3450faa7-0f7b-4974-b661-e821d966f764)